inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/fully_connected/fully_connected_kernel_mmad.cpp

```diff
@@ -90,9 +90,12 @@ FullyConnectedKernelMMAD::FullyConnectedTuningData FullyConnectedKernelMMAD::Get
     bool slm_div_factor_exception = input_batch == 300 && input_feature == 2048 &&
                                     output_batch == 300 && (output_feature == 324 || output_feature == 81);
 
+    size_t max_work_group_size = params.engineInfo.maxWorkGroupSize;
+    if (max_work_group_size > 256) max_work_group_size = 256;
```

Comment:
BTW, it makes sense:
- split `if` line on 2 lines for better reports from code coverage tools
- add "TODO" comment?